utilize_eye_in_hand_calibration.py

```diff
@@ -0,0 +1,53 @@
+"""
+Utilize the result of eye-in-hand calibration to transform (picking) point
+coordinates from the camera frame to the robot base frame.
+"""
+
+import numpy as np
+import cv2
+
+
+def _read_transform(file_name):
+    """Read transformation matrix from a YAML file.
+
+    Args:
+        file_name: Path to the YAML file.
+
+    Returns:
+        transform: Transformation matrix.
+
+    """
+
+    file_storage = cv2.FileStorage(file_name, cv2.FILE_STORAGE_READ)
+    transform = file_storage.getNode("PoseState").mat()
+    file_storage.release()
+
+    return transform
+
+
+def _main():
+
+    np.set_printoptions(precision=2)
+
+    # Define (picking) point in camera frame
+    point_camera = np.array([81.2, 18.0, 594.6, 1])
```

Comment:
Do you have a better name for this variable?

Reply:
point_camera_frame is this better?

Reply:
Unsure, I just think the name might be a bit confusing for people

Reply:
now it is: point_in_base_frame, point_in_camera_frame

Reply:
I changed transform poses names so that it is similar style